RELEASE-NOTES.txt

```diff
@@ -6,6 +6,7 @@
 -----
 - [**] Fixed Collect Payment from the menu on iPads running iOS 16 [https://github.com/woocommerce/woocommerce-ios/pull/13491]
 - [Internal] Added missing events for custom range on the stats dashboard card. [https://github.com/woocommerce/woocommerce-ios/pull/13506]
+- [Internal] Disable focus for order rows on iPadOS 16 and 16.1 to avoid crashes. [https://github.com/woocommerce/woocommerce-ios/pull/13512]
```

Comment:
It doesn't need to be internal, fixing crashes can/should be mentioned in release notes, even if we aren't specific about how/which one we fix.